apps/backend/package.json

```diff
@@ -16,9 +16,16 @@
   "dependencies": {
     "@nestjs/common": "^10.3.8",
     "@nestjs/core": "^10.3.8",
-    "@nestjs/mapped-types": "*",
+    "@nestjs/mapped-types": "2.0.5",
     "@nestjs/platform-express": "^10.3.8",
+    "@nestjs/swagger": "^7.3.1",
     "@prisma/client": "^5.13.0",
+    "class-transformer": "^0.5.1",
+    "class-validator": "^0.14.1",
+<<<<<<< HEAD
```

Comment:
Please resolve the merge conflict (just delete all between <<< and >>> 76...43).